nautobot/docs/index.md

```diff
@@ -57,6 +57,10 @@ Nautobot is built on the [Django](https://djangoproject.com/) Python Web framewo
 | Task queuing       | Redis/Celery/django-rq          |
 | Live device access | NAPALM                          |
 
+The low level diagram displays how data travels through Nautobot's application stack.
+
+![Application stack diagram](/docs/media/nautobot_apllication_stack_low_level.png "Application stack diagram")
```

Comment:
To me it's if anything more confusing to have multiple boxes with the same "Web Services" subheading, as it seems like they're redundant that way. What happened to the more specific subheadings ("HTTP Reverse Proxy", "WSGI HTTP Server", "Django Web Application", "Python Web Framework", etc.) that you had in your original draft that you'd shared on Slack?

Same comment on having three boxes labeled "Task Worker" and two boxes labeled "In-Memory Data Store".